nurseconnect/templates/profiles/done.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h4 class="heading heading--subtitle">
+<p>Hello</p>
+    {% trans "Thank you for joining!" %}
+</h4>
+<div class="profiles">
+  <p>{% trans "Let us know your birthday to get access to exclusive content." %}</p>
```

Comment:
um this is a new file according to git so as far as we are concerned you wrote it :) please fix things you touch ;)